src/schemas/ProgramDonorSummary/resolvers/summaryEntriesAndStats.ts

```diff
@@ -452,6 +452,23 @@ const programDonorSummaryEntriesAndStatsResolver: (esClient: Client) => DonorEnt
 				const dnaTNMatchedPairQuery = esb.boolQuery().should(shouldQueries);
 				queries.push(dnaTNMatchedPairQuery);
 			}
+
+			if (field === EsDonorDocumentField.validWithCurrentDictionary && filter.values.length > 0) {
+				for (const value of filter.values) {
+					if (value === 'VALID' || value === 'INVALID') {
+						queries.push(
+							esb
+								.boolQuery()
+								.must(
+									esb.termsQuery(
+										EsDonorDocumentField.validWithCurrentDictionary,
+										value === 'VALID' ? true : false,
```

Comment:
```suggestion
										value === 'VALID',
```

the ternary is redundant, a comparison will return the boolean value you want

Reply:
Classic bit of overthinking/not thinking there hahaha, thanks updated